@markpbaggett hey sorry for the slow response on this. Trying to get this building, things consistently stop with the following error:
```
default: Virtualenv location: /home/vagrant/.local/share/virtualenvs/vagrant-G6U0N2py
    default: Creating a Pipfile for this project...
    default: Launching subshell in virtual environment...
    default: Traceback (most recent call last):
    default:   File "/home/vagrant/.local/bin/pipenv", line 11, in <module>
    default:     sys.exit(cli())
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 764, in __call__
    default:     return self.main(*args, **kwargs)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 717, in main
    default:     rv = self.invoke(ctx)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 1137, in invoke
    default:     return _process_result(sub_ctx.command.invoke(sub_ctx))
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 956, in invoke
    default:     return ctx.invoke(self.callback, **ctx.params)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 555, in invoke
    default:     return callback(*args, **kwargs)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/decorators.py", line 64, in new_func
    default:     return ctx.invoke(f, obj, *args, **kwargs)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/click/core.py", line 555, in invoke
    default:     return callback(*args, **kwargs)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/cli/command.py", line 390, in shell
    default:     pypi_mirror=state.pypi_mirror,
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/core.py", line 2184, in do_shell
    default:     shell.fork_compat(*fork_args)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/shells.py", line 121, in fork_compat
    default:     c.interact(escape_character=None)
    default:   File "/home/vagrant/.local/lib/python3.5/site-packages/pipenv/vendor/pexpect/pty_spawn.py", line 761, in interact
    default:     mode = tty.tcgetattr(self.STDIN_FILENO)
    default: termios.error: (25, 'Inappropriate ioctl for device')
    default: Looking in indexes: https://test.pypi.org/simple/, https://pypi.org/simple
    default: Collecting repox
    default:   Downloading https://test-files.pythonhosted.org/packages/fc/4b/d9bf2c805400571f5547db987c10d62412c6602d2c5ce93c7cf909d3a32a/repox-0.0.2-py3-none-any.whl
    default: Requirement already satisfied: requests>=2.2.1 in /usr/lib/python3/dist-packages (from repox) (2.9.1)
    default: Collecting xmltodict>=0.11.0 (from repox)
    default:   Downloading https://files.pythonhosted.org/packages/42/a9/7e99652c6bc619d19d58cdd8c47560730eb5825d43a7e25db2e1d776ceb7/xmltodict-0.11.0-py2.py3-none-any.whl
    default: Installing collected packages: xmltodict, repox
    default: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/local/lib/python3.5/dist-packages/xmltodict.py'
    default: Consider using the `--user` option or check the permissions.
```

I haven't given the python-related shell scripts a close review, but it looks like a small change there will finish the install process?

--- a/scripts/python.sh
+++ b/scripts/python.sh
@@ -47,4 +47,4 @@ mkdir pyrepox_testing
 cd /home/pyrepox_testing
 pipenv --python 3.7 install
 pipenv shell
-pipenv install repox
+pip install repox

--- a/scripts/python.sh
+++ b/scripts/python.sh
@@ -47,4 +47,4 @@ mkdir pyrepox_testing
 cd /home/pyrepox_testing
 pipenv --python 3.7 install
 pipenv shell
-pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple repox
+pipenv install repox

